__tests__/app.test.js

```diff
@@ -106,7 +106,7 @@ describe("GET: /api/articles", () => {
       });
   });
   describe("article queries", () => {
-    test("should return the articles filtered by given topics", () => {
+    test("should return the articles filtered by given topics//", () => {
```

Comment:
There is another test you should have in here which is a Status 200 - valid topic query but there are no articles so response is an empty array.